code/js/modules/BaseController.js

```diff
@@ -219,43 +356,48 @@
         (this.selectors.pause && this.doc().querySelector(this.selectors.pause))
       ),
       canPlayNext: this.overridePlayNext || !!(this.selectors.playNext && this.doc().querySelector(this.selectors.playNext)),
+      canMute: this.canMute,
       canLike: !!(this.selectors.like && this.doc().querySelector(this.selectors.like)),
+      canSeek: this.canSeek,
+      canSetVolume: this.canSetVolume,
       hidePlayer: this.hidePlayer
     };
+    state.volume = this.getVolume();
+    return state;
```

Comment:
What is the benefit of this change? (assuming the above recommendation about inside the brace is accepted).

Reply:
I think this is an old artifact from a time when getVolume might not exist depending on the controller type. Since it is implemented in BaseController it should be inside the brace.

Reply:
Sorry I should clarify; what's the benefit of setting state as a var and then returning it, as opposed to just returning it (the way it was)?

Reply:
I didn't think this to the end😅 I just fixed it in 9aa070b